src/lib/docs/get-docs-nav-items.ts

```diff
@@ -0,0 +1,45 @@
+import {
+	DocsNavItem,
+	ProductData,
+	ProductSlug,
+	RootDocsPath,
+} from 'types/products'
+
+/**
+ * Given ProductData for a specific product,
+ * Return items to render in a "Documentation" navigation section.
+ */
+export function getDocsNavItems(
```

Comment:
`getDocsNavItems` essentially does the map of `rootDocsPaths` that we did previously.

For products like Terraform, where we don't show all `rootDocsPaths` in docs nav menus, it also supports using `currentProductData.docsNavItems`.